crates/contract/src/call.rs

```diff
@@ -282,21 +281,20 @@ impl<P: TempProvider, D: CallDecoder> CallBuilder<P, D> {
         self
     }
 
-    /// Sets the `gas` field in the transaction to the provided value
-    pub fn gas(mut self, gas: U256) -> Self {
-        self.request = self.request.gas_limit(gas);
-        self
-    }
```

Comment:
this should not be removed

Reply:
> ld not be removed

thanks for review; but actually it's not removed